content/blog/laravel-performance-monitoring-complete-apm-comparison-guide/index.md

```
@@ -59,12 +59,12 @@ public function index()
 
 ### Performance Impact Without Monitoring:
 
-```text
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)